pom.xml

```diff
@@ -1660,6 +1660,18 @@
                 <scope>test</scope>
                 <version>${version.org.wildfly.glow}</version>
             </dependency>
+            <dependency>
```

Comment:
We have hard constraints when adding dependencies like this one. Is it possible to achieve the same goal without adding them?

Reply:
Should be possible, I could manually compose the api urls and then use a http client to make the request. I'll change that accordingly.

Reply:
Thanks. That would be great and make things simpler. Well, I hope that SDK is only running some basic HTTP calls :)

Reply:
We'll see :-)

Reply:
Done.